flink-connector-jdbc/src/test/java/com/getindata/flink/connector/jdbc/table/JdbcFilterPushdownPreparedStatementVisitorTest.java

```diff
@@ -144,7 +144,8 @@ public void testSimpleExpressionPrimitiveType() {
                 new Object[] {"real_col > 0.5", "real_col > ?", new BigDecimal("0.5")},
                 new Object[] {
                     "double_col <= -0.3", "double_col <= ?", new BigDecimal("-0.3")
-                })
+                },
+                new Object[] {"description LIKE '_bcd%'", "description LIKE ?", "_bcd%"})
             .forEach(
                 inputs ->
                     assertSimpleInputExprEqualsOutExpr(
```

Comment:
Just a suggestion, maybe a more suitable name for a test `testExpressionWithNull` -> `testExpressionEqualsNull` ?

Reply:
It's a fork of flink repo, I won't change test names.